docs/guides/integration-mobile-overview.md

```diff
@@ -58,6 +49,108 @@ For details, refer to one of the following guides:
+
+| Scenario | Option | Integration Guide |
+| :--- | :--- | :--- |
+| You have access to <Link href="../ref-info/glossary-uid#gl-dii">DII</Link> (email address or phone number) on the client side/within the mobile app, and want to keep changes within your app only. | Client-side integration | [UID2 Client-Side Integration Guide for Mobile](integration-mobile-client-side.md) |
+| You have access to DII on the server side only and can do the necessary development to generate UID2 tokens server-side, or you are using a <Link href="../ref-info/glossary-uid#gl-private-operator">Private Operator</Link>. | Client-Server Integration | [UID2 Client-Server Integration Guide for Mobile](integration-mobile-client-server.md) |
+
+### Managing the UID2 Token
+
+The following table summarizes the mobile integration options for managing the UID2 token, with links to the documentation for each option.
+
+| UID2 Mobile Implementation Option | Documentation Link |
+| :--- | :---|
+| Prebid.js | Either of the following implementation guides:<ul><li>[UID2 Client-Side Integration Guide for Prebid.js](../guides/integration-prebid-client-side.md)</li><li>[UID2 Client-Server Integration Guide for Prebid.js](../guides/integration-prebid-client-server.md)</li></ul> |
+| UID2 Android SDK | SDK doc: [SDK for Android Reference Guide](../sdks/sdk-ref-android.md)<br/>Implementation Guide:<ul><li>[UID2 Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side.md)</li><li>[UID2 Client-Server Integration Guide for Mobile](../guides/integration-mobile-client-server.md)</li></ul> |
+| UID2 iOS SDK | SDK doc: [SDK for iOS Reference Guide](../sdks/sdk-ref-ios.md)<br/>Implementation Guide:<ul><li>[UID2 Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side.md)</li><li>[UID2 Client-Server Integration Guide for Mobile](../guides/integration-mobile-client-server.md)</li></ul> |
+
+[**GWH_SW01 on the above: is the standard Prebid.js implementation guide a correct link? Or should we just link to the optional Prebid.js section in the mobile implementation guide?**]
+
+### Sending the Token to the Bidstream
+
+There are many options for sending your UID2 tokens to the bidstream.
+
+The following table summarizes the options supported by UID2.
+
+| Scenario | Option | Integration Guide |
+| :--- | :--- | :--- |
+| You want to retrieve ads on Google | Secure Signals | [Google Ad Manager Secure Signals Integration Guide](integration-google-ss.md) |
+| You want to use Google GMA to integrate video, banner, interstitial, or native ads into your app | xxx | [UID2 GMA Plugin for Android Integration Guide](../guides/mobile-plugin-gma-android.md) or [UID2 GMA Plugin for iOS Integration Guide](../guides/mobile-plugin-gma-ios.md) |
```

Comment:
I don't think the Option column is needed as we are gonna repeat "Google GMA/IMA" anyway. 

I think as well as the plugin guide we should link to the integration guide GMA/IMA section
http://localhost:3006/docs/guides/integration-mobile-client-side#optional-uid2-gmaima-plugin-for-gam-secure-signals-integration
http://localhost:3006/docs/guides/integration-mobile-client-server#optional-uid2-gmaima-plugin-for-gam-secure-signals-integration

We should add the Prebid Mobile Option here ("If you want to use Prebid Mobile SDK and Prebid Server to send out ad requests) and point to the integration guide's Prebid Mobile section.

Reply:
Done. Broke out Android and iOS onto separate lines... otherwise it's a bit more complicated. See what you think.